### DISCUSSION TITLE
Is MIMO supported in srsUE zmq mode

### DISCUSSION THREAD

**Discussion by Zplusless** [srsran/srsRAN_Project#620] [General Help] [answered]
Hi, I cannot find any discussion on MIMO in ZMQ.

Since srsUE zmq mode supports a maximum bandwidth of 20 MHz, is it possible to acquire a higher brate using MIMO in zmq mode?

**Comment by Zplusless** [comment]
I have found some conflicting information, and it makes me confused. 

1. srsUE support MIMO: [Advanced Usage — srsRAN 4G 23.11 documentation](https://docs.srsran.com/projects/4g/en/latest/usermanuals/source/srsue/source/4_ue_advanced.html?highlight=mimo#mimo)

2. srsUE does not support MIMO: https://github.com/srsran/srsRAN_Project/discussions/109#discussioncomment-8422519

**Reply by pgawlowicz** [reply, marked as answer]
Hi, the srsUE contains both 4g and 5g versions, but MIMO is supported only in 4g version.

**Reply by Zplusless** [reply]
What a pity. Thanks for your help!

**Comment by anasalbakri** [comment]
So the problem is in srsue itself not the ZMQ? 
if I use just srsran gNB with ZMQ to send IQ samples over ethernet to another machine I can use MIMO right?

**Reply by Zplusless** [reply]
Hi @anasalbakri, do you find any alternative to SRSUE?

**Reply by anasalbakri** [reply]
> Hi @anasalbakri, do you find any alternative to SRSUE?

Hi @Zplusless  
Well, for now I'm limited to it but I might try with COTS UE. 
As for throughput did you get any better than 76 Mbps? for SISO, Bandwidth 20 MHZ, SCS 15 KHZ. 
I got maximum 76 Mbps with ZMQ and only 15 Mbps with USRP B210. I'm wondering if that's their limits.
